test/tools/libtesteth/BlockChainHelper.cpp

```diff
@@ -545,6 +545,22 @@ TestTransaction TestTransaction::defaultTransaction(u256 const& _nonce, u256 con
 	return TestTransaction(txObj);
 }
 
+TestTransaction TestTransaction::defaultZeroTransaction(u256 const& _gasLimit, bytes const& _data)
+{
+	json_spirit::mObject txObj;
+	txObj["data"] = toHex(_data);
+	txObj["gasLimit"] = toString(_gasLimit);
+	txObj["gasPrice"] = toString(0);
+	txObj["nonce"] = toString(0);
+	txObj["v"] = toString(1);
```

Comment:
this is just the test version. yes we could define an argument in this function for chainID
by default chainID must be = 1 in network test.